app/models/ability.rb

```diff
@@ -11,7 +11,7 @@ def initialize(user)
 #
     else
       can :read, User, id: user.id
-      can :read, Order, user_id: user.id
+      can :read, Order#, user_id: user.id  not sure if I should replace this with something else that identifies the user?
```

Comment:
I re-pushed without this comment in here, but it's still showing?
But more importantly, this is still a question I have. Is it ok to remove the user_id part of that and still have the permissions work the way they should?